deployment/6_register_modules.js

```diff
@@ -1,115 +1,109 @@
   await manager.setup();
 
   const { configurator } = manager;
-  const { deployer } = manager;
   const { versionUploader } = manager;
-  const { gasPrice } = deployer.defaultOverrides;
-
-  const deploymentWallet = deployer.signer;
-
   const { config } = configurator;
-  console.log("Config:", config);
-
-  const GuardianManagerWrapper = await deployer.wrapDeployedContract(GuardianManager, config.modules.GuardianManager);
-  const LockManagerWrapper = await deployer.wrapDeployedContract(LockManager, config.modules.LockManager);
-  const RecoveryManagerWrapper = await deployer.wrapDeployedContract(RecoveryManager, config.modules.RecoveryManager);
-  const ApprovedTransferWrapper = await deployer.wrapDeployedContract(ApprovedTransfer, config.modules.ApprovedTransfer);
-  const TransferManagerWrapper = await deployer.wrapDeployedContract(TransferManager, config.modules.TransferManager);
-  const TokenExchangerWrapper = await deployer.wrapDeployedContract(TokenExchanger, config.modules.TokenExchanger);
-  const NftTransferWrapper = await deployer.wrapDeployedContract(NftTransfer, config.modules.NftTransfer);
-  const CompoundManagerWrapper = await deployer.wrapDeployedContract(CompoundManager, config.modules.CompoundManager);
-  const MakerV2ManagerWrapper = await deployer.wrapDeployedContract(MakerV2Manager, config.modules.MakerV2Manager);
-  const RelayerManagerWrapper = await deployer.wrapDeployedContract(RelayerManager, config.modules.RelayerManager);
-  const VersionManagerWrapper = await deployer.wrapDeployedContract(VersionManager, config.modules.VersionManager);
-
-  const ModuleRegistryWrapper = await deployer.wrapDeployedContract(ModuleRegistry, config.contracts.ModuleRegistry);
-  const MultiSigWrapper = await deployer.wrapDeployedContract(MultiSig, config.contracts.MultiSigWallet);
+
+  const GuardianManagerWrapper = await GuardianManager.new(config.modules.GuardianManager);
+  const LockManagerWrapper = await LockManager.new(config.modules.LockManager);
+  const RecoveryManagerWrapper = await RecoveryManager.new(config.modules.RecoveryManager);
+  const ApprovedTransferWrapper = await ApprovedTransfer.new(config.modules.ApprovedTransfer);
+  const TransferManagerWrapper = await TransferManager.new(config.modules.TransferManager);
+  const TokenExchangerWrapper = await TokenExchanger.new(config.modules.TokenExchanger);
+  const NftTransferWrapper = await NftTransfer.new(config.modules.NftTransfer);
+  const CompoundManagerWrapper = await CompoundManager.new(config.modules.CompoundManager);
+  const MakerV2ManagerWrapper = await MakerV2Manager.new(config.modules.MakerV2Manager);
+  const RelayerManagerWrapper = await RelayerManager.new(config.modules.RelayerManager);
+  const VersionManagerWrapper = await VersionManager.new(config.modules.VersionManager);
+
+  const ModuleRegistryWrapper = await ModuleRegistry.new(config.contracts.ModuleRegistry);
+  const MultiSigWrapper = await MultiSig.new(config.contracts.MultiSigWallet);
```

Comment:
Shouldn't it be `ContractWrapper = await Contract.at(config...)` instead of using `ContractWrapper = await Contract.new(config...)`?

Reply:
Yes, thanks, updated.